testing/logging/test_reporting.py

```diff
@@ -49,6 +49,28 @@ def test_foo():
                                  'text going to stderr'])
 
 
+def test_root_logger_affected(testdir):
+    testdir.makepyfile('''
+        import logging
+
+        logger = logging.getLogger()
+        logger.addHandler(logging.FileHandler('pytest.log'))
```

Comment:
I would prefer to not add the FileHandler manually but use the --log-file cmd line option instead. I'm also fine with testing both cases

Reply:
Replaced with --log-file.